docs/remarkrc

```diff
@@ -16,21 +16,14 @@
         "maximum-heading-length": 60,
         "maximum-line-length": 80,
         "no-consecutive-blank-lines": true,
-        "no-duplicate-headings": true,
```

Comment:
@jeremywiebe could update the test fixtures to include an example that will fail / pass this rule?